packages/nimble-components/src/table-column/enum-text/group-header-view/index.ts

```diff
@@ -0,0 +1,47 @@
+import { DesignSystem } from '@microsoft/fast-foundation';
+import { observable } from '@microsoft/fast-element';
+import { TableGroupHeaderView } from '../../base/group-header-view';
+import { styles } from './styles';
+import { template } from './template';
+import type { ConvertedKeyMappingText } from '../../../mapping/text';
+import type { TableColumnEnumColumnConfig } from '../../enum-base';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-table-column-enum-text-group-header-view': TableColumnEnumTextGroupHeaderView;
+    }
+}
+
+/**
+ * A group header view for enum columns
+ */
+export class TableColumnEnumTextGroupHeaderView extends TableGroupHeaderView<
```

Comment:
Anything stopping us from extending `TableColumnTextGroupHeaderViewBase` for the enum text column? It would allow us to delete several tests for this column type.

Reply:
I thought I couldn't because the template wasn't exactly the same, but I realized I can very easily provide my own tweaked template while extending the base class.